coding/cpp_style_guide.rst

```diff
@@ -788,25 +788,26 @@ Keeps inputs together.
 
 .. _style-guide-cpp-3-34:
 
-3-34. Uncertainty values associated with a variable SHOULD be suffixed by one of ``Var``, ``Cov``, ``Sigma``.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-There is no universal suffix for uncertainties; i.e. no ``Err`` suffix will be used.
-The cases that we have identified, and their appropriate suffixes, are:
+3-34. Uncertainty values associated with a variable SHOULD be suffixed by one of ``Err`` or``Cov``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Standard deviation: ``Sigma`` (not ``Rms``, as ``rms`` doesn't imply that the mean's subtracted)
-- Covariance: ``Cov``
-- Variance: ``Var``
+Following the DPDD, we use ``Err`` (not ``Sigma``) to specify error quantities, as standard errors.
+``Sigma`` should be used to specify the inherent width of a distribution or function.
 
 .. code-block:: cpp
 
-   float xAstrom;          // x position computed by a centroiding algorithm
-   float xAstromSigma;     // Uncertainty of xAstrom
-   float yAstrom;
-   float yAstromSigma;
-   float xyAstromCov;
+    // for single measurements
+    float xCentroid;        // x position computed by a centroiding algorithm
+    float xCentroidErr;     // Uncertainty of xAstrom
+    float yCentroid;
+    float yCentroidErr;
+    float xyCentroidCov;    // Covariance of x/y centroid
+
+    // for distributions
+    float fpFluxMean;       // Weighted mean of forced-photometry flux (fpFlux)
+    float fpFluxMeanErr;    // Standard error of fpFluxMean.
+    float fpFluxSigma;      // Standard deviation of the distribution of fpFlux.
  
-The postfix ``Err`` can easily be misinterpreted as error flags.
```

Comment:
It's kind of funny that we end up having to remove this sentence and hope that `Err` is no longer confused for error flags.

Reply:
To be fair, I didn't even know what that statement meant. Don't we identify error flags with `Flag`?

Reply:
My comment was mainly that in the past someone was convinced that `Err` was a bad choice and now we have changed our minds.